### PR TITLE
Add methods to enable revolving carousel

### PR DESCRIPTION
**Pull request by seanperez29**

### CODE REVIEW

File: Example/Pods/Pods.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1008,7 +1005,7 @@
 				GCC_PREFIX_HEADER = "Target Support Files/Nimble/Nimble-prefix.pch";
 				INFOPLIST_FILE = "Target Support Files/Nimble/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```
Comment (justinvallely) [inline Example/Pods/Pods.xcodeproj/project.pbxproj]: Lets keep this as low as possible to achieve the revolving gallery goal. 9.0? Even less if possible.

File: Example/Pods/Pods.xcodeproj/project.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
+   <FileRef
+      location = "self:">
+   </FileRef>
```
Comment (justinvallely) [inline Example/Pods/Pods.xcodeproj/project.xcworkspace/contents.xcworkspacedata:6]: I'm not sure what this file is. Might need to be added to the git ignore. Was this a conscientious change?

File: Pod/Classes/SwiftPhotoGallery.swift [hunk 3]
```diff
@@ -401,6 +403,30 @@ extension SwiftPhotoGallery: UICollectionViewDataSource {
...
+        var cell = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionElementKindSectionFooter, withReuseIdentifier: "SwiftPhotoGalleryCell", for: indexPath) as! SwiftPhotoGalleryCell
+
+        switch kind {
+        case UICollectionElementKindSectionFooter:
+            cell.image = getImage(currentPage: 0)
+        case UICollectionElementKindSectionHeader:
+            cell = collectionView.dequeueReusableSupplementaryView(ofKind: UICollectionElementKindSectionHeader, withReuseIdentifier: "SwiftPhotoGalleryCell", for: indexPath) as! SwiftPhotoGalleryCell
+            cell.image = getImage(currentPage: numberOfImages - 1)
+        default:
+            assert(false, "Unexpected element kind")
```
Comment (justinvallely) [inline Pod/Classes/SwiftPhotoGallery.swift]: Preferred syntax is: `assertionFailure("Unexpected element kind")`

File: Example/Pods/Nimble/Sources/Nimble/Matchers/AllPass.swift [hunk 4]
```diff
@@ -65,7 +65,7 @@ extension NMBObjCMatcher {
             
             var collectionIsUsable = true
             if let value = actualValue as? NSFastEnumeration {
-                let generator = NSFastEnumerationIterator(value)
+                var generator = NSFastEnumerationIterator(value)
```
Comment (justinvallely) [inline Example/Pods/Nimble/Sources/Nimble/Matchers/AllPass.swift]: This change will get overwritten by a pod install. Update Nimble from the example project workspace's pod file and the run `pod update` to fix this.

File: Example/Pods/Pods.xcodeproj/project.pbxproj (hunk 1 above)
Comment (justinvallely) [reply Example/Pods/Pods.xcodeproj/project.pbxproj]: We'll also need to update the podspec to reflect this change as well.

**Review by justinvallely — CHANGES_REQUESTED** [review]
Since this is in the public space, some people might prefer to toggle this feature on/off. Please add a public boolean to enable/disable this feature following the syntax on line 93 of SwiftPhotoGallery.swift. Let's default it to enabled though!

**Comment by seanperez29** [comment]
I like that option to toggle the carousel feature on/off. I thought about it earlier and was figuring out the best place to hide the footer and header. I just added that feature, and wrapping up the other stuff as well right now.

File: Example/Pods/Pods.xcodeproj/project.xcworkspace/contents.xcworkspacedata (hunk 2 above)
Comment (seanperez29) [reply Example/Pods/Pods.xcodeproj/project.xcworkspace/contents.xcworkspacedata:6]: This wasn't a conscientious change, not sure what caused that.

File: Example/Pods/Pods.xcodeproj/project.pbxproj (hunk 1 above)
Comment (seanperez29) [reply Example/Pods/Pods.xcodeproj/project.pbxproj]: 9.0 will be the lowest, just updated it

**Review by justinvallely — APPROVED** [review]
Nice work! Thanks for adding this.

**Pull request merged**